bigbluebutton-html5/imports/api/audio/client/bridge/livekit.ts

```
@@ -0,0 +1,352 @@
+          errorMessage: (error as Error).message,
+          errorName: (error as Error).name,
+          errorStack: (error as Error).stack,
+          bridgeName: this.bridgeName,
+          role: this.role,
+        },
+      }, 'Failed to publish audio track');
+      throw error;
+    }
+  }
+
+  private unpublish(): Promise<void | (void | LocalTrackPublication | undefined)[]> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)